docs/modules/rtcmem.md

```diff
@@ -0,0 +1,64 @@
+# RTC User Memory Module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2015-06-25 | [DiUS](https://github.com/DiUS), [Johny Mattsson](https://github.com/jmattsson) | [Johny Mattsson](https://github.com/jmattsson) | [rtcmem.c](../../app/modules/rtcmem.c)|
+
+The rtcmem module provides basic access to the battery backed memory.
```

Comment:
I'd suggest updating this to reflect what the IDF says:

RTC Slow memory[](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/memory-types.html#rtc-slow-memory)

_Global and static variables used by code which runs from RTC memory must be placed into RTC Slow memory. For example [deep sleep](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/deep-sleep-stub.html) variables can be placed here instead of RTC FAST memory, or code and variables accessed by the [ULP Coprocessor programming](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/system/ulp.html)._

(but maybe avoid dragging fast rtc mem into the picture?)